specification/src/main/asciidoc/webprofile/Introduction.adoc

```diff
@@ -86,10 +90,10 @@ technology or combinations of technologies.
 Let’s look at some examples of requirements
 from each grouping.
 
-For the first one, the Java EE Platform
+For the first one, the Jakarta EE Platform
 specification mandates support for the “ _java:_ ” naming context in all
 profiles. Consequently, Web Profile products must support it. For a
-similar reason, all Web Profile 8 products must support the Java Platform, Standard Edition 8 API.
+similar reason, all Web Profile 8 products must support the Java(TM) Platform, Standard Edition 8 API.
```

Comment:
(TM) vs ™

Reply:
Explained above.  The (TM) is rendered as ™.